### ISSUE TITLE
Searching Algorithm

### ISSUE THREAD

**Issue by theshoonyam** [krsh123/DSA-CPP#23]
This repository lacks searching algorithms. It would be great if it has at least popular searching algorithm.

**Comment by krsh123** [comment]
Yes, thank you for showing interest. I am assigning it you. Make a folder of Name Searching Algorithm
